### DISCUSSION TITLE
[match] creates certificates with country US

### DISCUSSION THREAD

**Discussion by csageder** [fastlane/fastlane#17000] [Q&A]
Hi,
I am new to fastlane and match, but I recognised that the certificates created by math have always country US which is wrong for me. Is there any possibility to configure match to set an other country when creating the CSR? I didn't found anything in the documentation. I even checked the code. For me it seams this is not foreseen. Did I have overseen something? 

Thanks for your support

**Comment by jack828** [comment]
I believe this is also causing an issue for me wherein I cannot use Apple Pay on releases.

Hope someone knows what do to here!

EDIT: I was doing something wrong in my stripe library, not related to this.
